src/registry_schemas/schemas/continuation_out.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/continuation_out",
+    "title": "Continuation Out Application Schema",
+    "definitions": {},
+    "required": [
+        "continuationOut"
+    ],
+    "type": "object",
+    "properties": {
+        "continuationOut": {
+            "type": "object",
+            "required": [
+                "details",
+                "continuationOutDate",
+                "foreignJurisdiction"
+            ],
+            "properties": {
+                "details": {
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 2000,
+                    "description": "A brief note to explain the continue out."
+                },
+                "continuationOutDate": {
+                    "type": "string",
+                    "format": "date",
+                    "title": "Effective date of Continuation Out."
+                },
+                "foreignJurisdiction": {
+                    "type": "object",
+                    "required": [
+                        "country",
+                        "legalName"
+                    ],
+                    "properties": {
+                        "country": {
+                            "type": "string"
+                        },
+                        "region": {
+                            "type": [
+                                "string",
+                                "null"
+                            ],
+                            "maxLength": 2,
+                            "title": "The foreign jurisdiction if country is Canada or USA."
+                        },
```

Comment:
so I guess the idea is that legal api would validate a valid region is provided when the `country` value provided is "Canada" or "US"?

Reply:
yes

Reply:
I think the region title is incorrect.

I think that:
- if country=CA then region can be one of the provinces or "Federal"
- if region=Other then country can be any country other than CA

Ref: https://preview.uxpin.com/e13906bf0d12d45f7c3ecb0b12e580b512fdfbaf#/pages/162653861/simulate/no-panels

Do you want to limit the values for region?

Reply:
We should look at a few examples:
1. continue out to Alberta
2. continue out to Federal
3. continue out to another country (eg, USA)

Reply:
Mihai mentioned to me today morning that this design has to be changed to ask Country first and region (if CA, US)

Reply:
That makes more sense to me :)  The UI can start with Canada as a default.

Should we keep track of US states in the region field? I didn't see that in the uxpin.

What will region be for another country?

Reply:
We need to keep track of Canada province and US states, for other countries region is not needed (probably hide that field)

Reply:
OK. Whatever happens, I'd like to see a few examples 😃

Reply:
Do  you have more examples (in Canada or US, not in Canada or US)?

Reply:
Sample JSON for the different cases, and maybe unit tests to go along with it.

Reply:
Included examples in region. Schema won't be validating region or country, will include unit test in legal-api validation ticket.

Reply:
I was thinking about a working example in src/registry_schemas/example_data/schema_data.py.

Thanks for the comment.